h/search/__init__.py

```diff
@@ -38,24 +35,11 @@ def includeme(config):
 
     connect(hosts=[settings['es.url']], **kwargs)
 
-    # Connection to old (ES1.5) follows
-    settings.setdefault('es.host', 'http://localhost:9200')
-    settings.setdefault('es.index', 'hypothesis')
```

Comment:
Remove the default initialization of `es.index` here breaks the `hypothesis init` command.

Reply:
Oops that shouldn't have been removed. I fix.